eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -1,5 +1,6 @@
 parameters:
   PackageName: ""
+  PackagePath: ""
```

Comment:
Where does `PackagePath` come from? If we want to just pass the service name here instead, we can use the `rush-runner` script to run `execute:samples`.

Reply:
As an alternative, you could eliminate this new parameter and just base it off the `PackageName` using the `npm-run-project` script, eg:

```
node common/scripts/npm-run-project.js "${{parameters.PackageName}}" execute:samples

Reply:
Thanks! this works perfect!